tests/test_validators.py

```diff
@@ -38,3 +39,23 @@ def test_number(self):
 
         with self.assertRaises(ValidationError):
             validator('spam')
+
+
+class FontTests(TestCase):
+    def test_font_family_name(self):
+        validator = is_font_family
+        invalid_cases = [
+            'Red/Black, sans-serif',
+            '"Lucida" Grande, sans-serif',
+            'Ahem!, sans-serif',
+            'test@foo, sans-serif',
+            '#POUND, sans-serif',
+            'Hawaii 5-0, sans-serif',
+            '123',
+        ]
+        for case in invalid_cases:
+            with self.assertRaises(ValidationError):
+                validator(case)
```

Comment:
As a stylist thing - why rename is_font_family to validator for this use case?

Reply:
Hmmmm I had two separate validators and then turned them into one, this is a leftover of that. Will fix